### DISCUSSION TITLE
키글챌 주간 통계 자동화 프로그램 제작기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#147] [1. 개발 관련 글 모음]
미루고 미루던 자동화 프로그램을 드디어 만들었습니다...
이제 주간 통계가 월요일 00시마다 자동으로 올라갑니다!
Github GraphQL API가 정말 잘 되어 있더라구요
이거 때문에 하루종일 삽질했지만 어떻게 완성은 했으니 만족합니다^^7

다음에는 또 여유가 나면 이걸 고도화해서 멤버 별 작성 게시글 이력을 스프레드 시트에 작성하고, 반성문까지 트래킹할 수 있게 만들어볼 예정입니다.

이제 제대로 동작하는지 확인하려면 월요일까지 기다려야겠네요.
부디 제대로 돌아가기를!

[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/Github%20Discussion%20%EC%A3%BC%EA%B0%84%20%ED%86%B5%EA%B3%84%20%EC%9E%90%EB%8F%99%ED%99%94%20%EC%8B%9C%EC%8A%A4%ED%85%9C%20%EA%B5%AC%EC%B6%95%ED%95%98%EA%B8%B0%20-%201/)

**Comment by shkisme** [comment]
와! 자동화!
수고하셨습니당 👏

**Reply by 02ggang9** [reply]
신기하네요~!! 슬랙이랑 연동시키고 싶은 기능이 있는데 잘 참고하겠습니다~
수고하셨습니다..!!
